src/haconiwa/scan/scanner.py

```
@@ -0,0 +1,342 @@
+                for i, line in enumerate(lines):
+                    if regex.search(line):
+                        match_info = {
+                            'file': str(file_path.relative_to(self.base_path)),
+                            'line_number': i + 1,
+                            'line': line.strip(),
+                            'context': self._get_context(lines, i, context_lines)
+                        }
+                        results['matches'].append(match_info)
+                        results['total_matches'] += 1
+            
+            except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)